src/Types/Workout.ts

```diff
@@ -0,0 +1,114 @@
+import { Resources } from './Resources';
+export type WorkoutObjectType =
+  | 'exuipment'
+  | 'position'
+  | 'exercise'
+  | 'assignment'
+  | 'set'
+  | 'workout';
+
+export type Reference<T> = string | T;
+
+export interface WorkoutObject {
```

Comment:
Can we rename this? Calling it a WorkoutObject makes it hard to remember what it's doing. I suggest `ElementMetaData` but any name that describes the contents well will work

Reply:
Also include here dos and donts, description,